cli/integration_tests/turbo_help.t

```diff
@@ -73,3 +73,84 @@ Test help flag
         --version             version for turbo
   
   Use "turbo [command] --help" for more information about a command.
+
```

Comment:
Just FYI I'm also testing help's output in #2379.

Reply:
I saw that. I prefer prysk/cram for tests that try to match the UI since it allows for easy updating of the snapshots using the `-i` flag